- Fix path to /usr/local/share

- Don't fail if unrecognised autoconf command-line flags are present
(avoid problem with --disable-gtk-doc)

- Permit building if security/openpam is installed.

- Don't fail if pkg-config --list-all returns an error code.

- gnome-open -> xdg-open

- Don't pick up FFmpeg (just used for recanim, but linked into many binaries)

Index: configure
--- configure.orig
+++ configure
@@ -2855,11 +2855,6 @@ ac_config_headers="$ac_config_headers config.h"
 echo "current directory: `pwd`"
 echo "command line was: $0 $@"
 
-if ! test -z "$ac_unrecognized_opts" ; then
-  echo "" >&2
-  exit 2
-fi
-
 ###############################################################################
 #
 #       Autoheader stuff
@@ -15261,9 +15256,9 @@ fi
 
 # Check for the various Gnome help and URL loading programs.
 #
-WITH_BROWSER=gnome-open
+WITH_BROWSER=xdg-open
 if test "$have_gtk" = yes; then
-  for ac_prog in gnome-open
+  for ac_prog in xdg-open
 do
   # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
@@ -20652,7 +20647,7 @@ fi
 fi
 
 have_ffmpeg=no
-if test "$have_avformat" = yes -a \
+if false && test "$have_avformat" = yes -a \
         "$have_avcodec" = yes -a \
         "$have_avutil" = yes -a \
         "$have_swscale" = yes -a \
@@ -22816,7 +22811,6 @@ else
     warn2 "The errors are:"
     echo ''
     echo "$pkgerr" | sed 's/^/             > /g'
-    CONF_STATUS=1
   fi
 fi
 
@@ -23105,7 +23099,6 @@ elif test "$have_pam" = no -a "$enable_locking" = yes 
   if test -d /etc/pam.d -o -f /etc/pam.conf ; then
     warn  "Your system seems to have PAM, but PAM is not being used."
     warn2 "That is probably not going to work out well."
-    CONF_STATUS=1
   fi
 fi
 
